pronto/static/js/index.js

```diff
@@ -558,6 +570,17 @@ async function getRecentEntries() {
 
     document.getElementById('news-summary').innerHTML = text;
     document.querySelector('.item[data-tab="news"] .label').innerHTML = data.results.length.toString();
+
+    //list of curators
+    var dynamicSelect = document.getElementById('curators');
+    const curators = new Set(data.results.map((e => e.user)));
```

Comment:
Sort the curators by name:

```suggestion
    const curators = [...new Set(data.results.map((e => e.user)))].sort();
```